recipes/r-tinkr/recipe.yaml

```diff
@@ -0,0 +1,72 @@
+schema_version: 1
+
+context:
+  name: tinkr
+  version: "0.2.0"
+
+package:
+  name: r-${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://cran.r-project.org/src/contrib/tinkr_${{ version }}.tar.gz
+  sha256: f8b7823e4b97822b576660c2535fdf753e249120067f8394695ef10714a91e19
+
+build:
+  number: 0
+  merge_build_and_host_envs: true
```

Comment:
In conda-build syntax this used to be `merge_build_host` which I know think for `rattler-build` is `merge_build_and_host_envs`.